includes/collection/class-followers.php

```diff
@@ -404,4 +404,94 @@ public static function get_inboxes( $user_id ) {
 
 		return array_filter( $results );
 	}
+
+	/**
+	 * Get all Followers that have not been updated for a given time
+	 *
+	 * @param enum $output     The output format, supported ARRAY_N, OBJECT and ACTIVITYPUB_OBJECT.
+	 * @param int  $number     Limits the result.
+	 * @param int  $older_than The time in seconds.
+	 *
+	 * @return mixed The Term list of Followers, the format depends on $output.
+	 */
+	public static function get_outdated_followers( $output = ARRAY_N, $number = 50, $older_than = 604800 ) {
```

Comment:
I am not a big fan of the variable output dependent on the parameter (same for the next function). What's the benefit? Is it something you couldn't do differently? For example, I see that there is a normal array and an `ACTIVITYPUB_OBJECT` which is actually not an object but an array as well. If you added a `__toString()` function to `Follower` that returns the name, it would do the same as `ARRAY_N`.

For the other use case you could just by convention use something like this (you still need to expose the term inside Follower)

```
$outdated_followers = Followers::get_outdated_followers();
$outdated_terms = array_map( $outdated_followers, 'get_term' );
```

If you don't want to do the `__toString()`, you could also use this construct:
```
$outdated_followers = Followers::get_outdated_followers();
$outdated_terms = array_map( $outdated_followers, 'get_name' );
```